azure-pipelines.yml

```diff
@@ -194,3 +194,24 @@ jobs:
 
   - script: python setup.py sdist
     displayName: 'Build sdist'
+
+
+- job: DocsTest
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
+    displayName: Add conda to PATH
+  - script: conda env create -n test_env --quiet --file ci/doc.yml
```

Comment:
`requirements-py38.yml`?

Reply:
This has been removed, now only dealing standard Python/pip.